resources/definitions/tinyboy_mago.def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": 2,
+    "name": "mago 3d printer",
```

Comment:
I can't find anything about this Mago printer. Is there a specsheet online somewhere?

Reply:
https://www.magoprinter.com/mago-3d-printer in trial stage, and removed until release, thanks a lot ...